website/docs/getting-started/editor-setup.mdx

```diff
@@ -4,20 +4,19 @@ description: "Setting your code editor"
 ---
 
 :::important contribute
-This document only contains information for adding supporting in Jetbrains IDEs and VS Code.
-Feel free to contribute to add instructions for your editor of choice.
+Using a different editor? Feel free to add instructions for your editor of choice.
 :::
 
 ## Add a template for creating components
 
-### Jetbrains IDEs
+### JetBrains IDEs
 
 1. Navigate to File | Settings | Editor | Live Templates.
-2. Select Rust and click on + icon to add a new Live Template.
+2. Select Rust and click on the + icon to add a new Live Template.
 3. Give it a name and description of your preference.
-4. Paste the following snippet in Template Text section
+4. Paste the following snippet in Template Text section.
 5. Change the applicability on the lower right, select Rust > Item > Module
-```rust ,ignore
+```rust, ignore
```

Comment:
I believe the space before the comma is necessary.

Reply:
Made the change as requested.

I'm not sure whether the space matters. I've seen it with no, i.e., ```` ```rust,ignore````
- https://stackoverflow.com/questions/42348075/how-to-put-a-line-into-the-documentation-which-is-ignored-for-doc-tests

I could make another PR to remove the space across the project if desired.

Reply:
The problem here isn't rustdoc, it's Docusaurus. Without the space it doesn't highlight the code at all. If we were talking about rustdoc code snippets then there shouldn't be a space there.